LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 ScrapingHub
+Copyright (c) 2021 Zyte Group Ltd
```

Comment:
I'm not sure if you can just fix a copyright in this way. I usually saw a list of owners. Even if a company was renamed, I've seen how it was mentioned in a license file:

Copyright (c) 2018-2021 ScrapingHub
Copyright (c) 2021 Zyte Group Ltd

It worth to clarify this one

Reply:
This is a good point, thanks Sergey. I will check with legal to understand the best solution.